### DISCUSSION TITLE
Equation, tables and Figures numbering

### DISCUSSION THREAD

**Discussion by VEZY** [fonsp/Pluto.jl#1409] [Questions about using Pluto]
Everything is in the title. I'm used to Rmarkdown, but I'm wondering whether it is doable in Pluto too?

**Comment by disberd** [comment]
Not really but it could be done somehow, there is some discussion of a hacky implementation for equations only on discourse in [this](https://discourse.julialang.org/t/pluto-how-to-number-equations-in-latex-mode/59900/8) thread.

**Reply by VEZY** [reply]
Thanks! I'll have a look a it.
